packages/mermaid/src/docs/syntax/sequenceDiagram.md

```diff
@@ -130,6 +130,15 @@ sequenceDiagram
     Note over Alice,John: A typical interaction
 ```
 
+It is also possible to add a line break (applies to text input in general):
+
+```mermaid-example
+sequenceDiagram
+    Alice->John: Hello John, how are you?
+    Note over Alice,John: A typical interaction<br>But now in two lines
+```
+
+
```

Comment:
```suggestion
```

I think prettier is throwing an error due to the double empty newline.

This should be automatically fixed via `pnpm run lint:fix`